.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description
+
+Please provide a brief description of the changes made in this pull request.
+
+## Checklist
+
+Please make sure that the following checkpoints have been completed before submitting this pull request:
+
+- [ ] Have you followed the code convention guidelines specified in the repository?
+- [ ] Have you included the updated method/function documentation in the `documentation.md` file?
+
+## Code Convention
+
+Please provide a brief explanation of how your code adheres to the code convention guidelines in the repository.
+
+## Documentation
+
+Please provide a brief explanation of how your changes have been documented in the `documentation.md` file.
+
+## Additional Information
+
+Please include any additional information or resources that may be relevant to this pull request.
```

Comment:
Remove all the rest parts and include one section. That is, upload a screenshot demonstrating the method you have created.